js/localStorage.js

```diff
@@ -0,0 +1,22 @@
+export default function setInfo() {
+  const name = document.getElementById('name').value;
+  const email = document.getElementById('email').value;
+  const coments = document.getElementById('comment').value;
+  const info = {
+    name,
+    email,
+    coments,
+  };
+  localStorage.setItem('formInfo', JSON.stringify(info));
+}
+
+export function getInfo() {
```

Comment:
- [ ] Kindly consider using ES6 arrow function method instead of the function declaration that you are using right now. and kindly apply this in all of the instances where you are using regular function declaration. all this is because we should follow ES6 syntax to follow best practices.

Reply:
done